From 6048b61590bd8721964f3ef896a9fdfe1c388ec9 Mon Sep 17 00:00:00 2001
From: Florin Sarbu <florin@balena.io>
Date: Thu, 9 Jul 2020 13:19:34 +0200
Subject: [PATCH] arm64: dts:: fsl-imx8mm-var-dart: Disable CQE for the eMMC

We have observed CQE presents some stability issues on a customer board
so let's disable it.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-var-dart.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-var-dart.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-var-dart.dts
index 02c6291..e22cf1ee 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-var-dart.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-var-dart.dts
@@ -1001,6 +1001,7 @@
 	pinctrl-2 = <&pinctrl_usdhc3_200mhz>;
 	bus-width = <8>;
 	non-removable;
+	no-cqe;
 	status = "okay";
 };
 
-- 
2.7.4

